src/manifold/include/manifold.h

```diff
@@ -65,8 +66,13 @@ class Manifold {
   static Manifold Extrude(Polygons crossSection, float height,
                           int nDivisions = 0, float twistDegrees = 0.0f,
                           glm::vec2 scaleTop = glm::vec2(1.0f));
+  static Manifold Extrude(Clippoly crossSection, float height,
```

Comment:
I think we can remove the `Polygon` versions - better to force the input to not have overlaps.

Reply:
Yea, nice to have the abstraction along the way to prevent avoidable errors.